incubator/mariadb-galera/README.md

```diff
@@ -0,0 +1,115 @@
+# MariaDB Galera Cluster
```

Comment:
I've used the one from the mariadb charts as a template, basics really are the same.